site/docs/guides/basic-drawing.md

```diff
@@ -652,3 +756,59 @@ fn pixel(x: i32, y: i32) void {
 ```
 
 </MultiLanguageCode>
+
+For better control, we can also write functions to get and set pixels in a more direct way:
```

Comment:
To me this seems too similar to the existing pixel drawing function example on this page (under Direct Framebuffer Access). I see that this version is more detailed, though the main purpose of the `pixel()` is to introduce how the framebuffer works.

Reply:
I think we could get rid of `pixel()` and use these new functions instead. The `pixel()` function was very hard for me to understand at first and didn't do much other than just setting a pixel on a given location.

Although is not as frequent, there's always someone who casually asks about how to use the framebuffer, only to find this section and get confused. I will even risk telling that this section is the main cause of why users always request for having a set/get pixel function into the default API.

So... what do you think?